package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "ejs": "2.0.8",
-    "exif": "git://github.com/cianclarke/node-exif.git",
+    "exif": "^0.4.0",
```

Comment:
Not sure if https://github.com/cianclarke/node-exif/commit/7f61a74c820eed3e010c278d8d7003009b379d48 got fixed in the core branch of node-exif, so leaving this open to remind me to verify when I get back from my trip abroad on Tuesday!